charm/kafka/reactive/storage.py

```diff
@@ -9,6 +9,28 @@
 from charmhelpers.core.hookenv import log
 
 
+def create_or_get_brokerid(log_dir):
+    broker_path = os.path.join(log_dir, '.broker_id')
+    storageids = hookenv.storage_list('logs')
+    broker_id = storageids[0].split('/')[1]
+
+    if os.path.exists(broker_path):
+        with open(broker_path, 'r') as f:
+            try:
+                broker_id = int(f.read().replace('\n', ''))
```

Comment:
f.read().strip() would be a more robust way to strip off leading and trailing whitespace.